.github/workflows/remark.yml

```
@@ -24,7 +24,7 @@ jobs:
         node-version: '18.x'
 
     - name: Install remark
-      run: npm install remark-cli remark-lint remark-lint-maximum-line-length remark-preset-lint-recommended remark-gfm
+      run: npm ci
```

I don't really like having `package.json` in the repo just for this. Couldn't you change the `npm install` command to add the version. 
```suggestion
      run: npm install remark-cli remark-lint remark-lint-maximum-line-length@"^3.1.3" remark-preset-lint-recommended remark-gfm
```

---

I didn't think mixing old versions of lints with the new `remark-lint` would work but it seems to be fine so I'll use that